blog/announcements.xml

```diff
@@ -2,6 +2,115 @@
+      </a>
+      <p>
+        Hey everyone, we're <a href="https://github.com/nrdxp">Timothy DeHerrera</a> and <a href="https://github.com/tomberek">Tom Bereknyei</a>,
+        the release managers for 21.11. As promised, the latest stable
+        release is here: NixOS 21.11 “Porcupine”.
+      </p>
+
+      <ul>
+        <li>
+          <a href="/manual/nixos/stable/release-notes.html#sec-release-21.11">Release manual</a>
+          <ul>
+            <li><a
+                href="/manual/nixos/stable/release-notes.html#sec-release-21.11-highlights">Highlights</a>
+            </li>
+            <li><a href="/manual/nixos/stable/release-notes.html#sec-release-21.11-new-services">New
+                Services</a></li>
+            <li><a
+                href="/manual/nixos/stable/release-notes.html#sec-release-21.11-incompatibilities">Backward
+                Incompatibilities</a></li>
+            <li><a
+                href="/manual/nixos/stable/release-notes.html#sec-release-21.11-notable-changes">Other
+                Notable Changes</a></li>
+          </ul>
+        </li>
+        <li><a href="/manual/nixos/stable/index.html#sec-upgrading">Upgrade Instructions</a></li>
+        <li><a href="/download.html#download-nixos">Image downloads</a></li>
+      </ul>
+
+      <p>
+        The 21.11 release was possible due to the efforts of <strong>1541</strong>
+        contributors in <strong>41960</strong> commits. We would especially
+        like to thank our top 10 contributors:
+        <a href="https://github.com/supersandro2000">Sandro Jäckel</a>,
+        <a href="https://github.com/fabaff">Fabian Affolter</a>,
+        <a href="https://github.com/mweinelt">Martin Weinelt</a>,
+        <a href="https://github.com/figsoda">figsoda</a>,
+        <a href="https://github.com/Artturin">Artturin</a>,
+        <a href="https://github.com/marsam">Mario Rodas</a>,
+        <a href="https://github.com/bobby285271">Bobby Rong</a>,
+        <a href="https://github.com/mic92">Jörg Thalheim</a>,
+        <a href="https://github.com/dotlambda">Robert Schütz</a>,
+        <a href="https://github.com/primeos">Michael Weiss</a>.
+      </p>
+
+      <p>
+        NixOS is already known as
+        <a href="https://repology.org/repositories/statistics/newest">the most up to date distribution</a>
+        and is
+        <a href="https://repology.org/repositories/statistics/total">in the top three by total number of packages</a>.
+      </p>
+
+      <ul>
+        <li>XXX 12985 packages were added</li>
+        <li>XXX 14109 packages were removed</li>
+        <li>XXX 16768 packages were updated</li>
+      </ul>
+
+      <p>
+        Likewise, our NixOS module system got bigger and better:
+      </p>
+
+      <ul>
+        <li>XXX 1527 options were added</li>
+        <li>XXX 290 options removed</li>
+        <li>XXX 400 options were updated</li>
+      </ul>
```

Comment:
this

Reply:
@garbas, if I was informed correctly, we usually get these number's from elastisearch, is that something you can assist us with?